arm/ripper/logger.py

```diff
@@ -23,9 +23,22 @@ def setuplogging(job):
         logfile = job.label + ".log"
 
     if cfg['LOGPATH'][-1:] == "/":
-        logfull = cfg['LOGPATH'] + logfile
+        #Check to see if file already exists, if so, create a new file
+        ilogfull = cfg['LOGPATH'] + logfile
```

Comment:
More verbosity in variable names would be good, I don't understand what ilogfull means.